setup.py

```diff
@@ -12,7 +12,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import os
 import setuptools
 
 setuptools.setup(
```

Comment:
Can you bump version to 2.4.0?

It's currently not handled automatically on merges :/

Reply:
Pushing the update now